Lib/site.py

```diff
@@ -409,6 +420,9 @@ def setcopyright():
         builtins.credits = _sitebuiltins._Printer(
             "credits",
             "Jython is maintained by the Jython developers (www.jython.org).")
+    elif is_pypy:
+        credits = "PyPy is maintained by the PyPy developers: http://pypy.org/"
+        license = "See https://foss.heptapod.net/pypy/pypy/src/default/LICENSE"
```

Comment:
This doesn’t seem correct, unless I’m not reading the code correctly.

Either define `_Printer` objects for `builtins.credits` and `builtins.license`, or change the code below to use the `credits` and `license` strings defined here.

Reply:
fixing.